.circleci/defaults.sh

```diff
@@ -47,7 +47,7 @@ LOCAL_FIRESIM_DIR=$LOCAL_CHIPYARD_DIR/sims/firesim/sim
 
 # key value store to get the build groups
 declare -A grouping
-grouping["group-cores"]="chipyard-ariane chipyard-rocket chipyard-hetero chipyard-boom chipyard-sodor"
+grouping["group-cores"]="chipyard-ariane chipyard-rocket chipyard-hetero chipyard-boom chipyard-sodor chipyard-digitaltop"
```

Comment:
Is this tested? I don't see a corresponding `.circleci` file change?

Reply:
I don't want to test if the simulator runs, I only want to test if it compiles. 
Thus the config file does not need to be changed, since every config in group-cores is built.